tests/catch/CMakeLists.txt

```diff
@@ -49,9 +53,10 @@ if(UNIX)
     set(CMAKE_CXX_COMPILER "${HIP_PATH}/bin/hipcc")
     set(CMAKE_C_COMPILER "${HIP_PATH}/bin/hipcc")
     set(HIPCONFIG_EXECUTABLE "${HIP_PATH}/bin/hipconfig")
-    execute_process(COMMAND perl ${HIPCONFIG_EXECUTABLE} --version
+    execute_process(COMMAND ${HIPCONFIG_EXECUTABLE} --version
```

Comment:
you don't need to invoke the perl command explicitly as far as I know. This should allow for using both HIPCC and Perl based hipcc